code/modules/shuttle/medisim.dm

```diff
@@ -0,0 +1,55 @@
+///These are for the medisim shuttle
+
+/obj/machinery/capture_the_flag/medisim
+	game_area = /area/shuttle/escape/simulation
+
+/obj/machinery/capture_the_flag/medisim/spawn_team_member(client/new_team_member)
+	var/mob/living/carbon/human/M = ..()
+	M.remove_all_languages(LANGUAGE_CTF)
+	M.grant_language(/datum/language/monkey, TRUE, TRUE, LANGUAGE_CTF)
+	randomize_human(M)
+	var/oldname = name
+	M.name = "[M.gender == MALE ? "Sir" : "Lady"] [oldname]"
```

Comment:
```suggestion
	var/title
	switch (M.gender)
		if (MALE)
			title = "Sir"
		if (FEMALE)
			title = "Lady"
		else
			title = "Noble"
	M.name = "[title] [oldname]"
```

We have gender neutral characters, Noble is the best I could come up with.

Reply:
the counterpart to sir is dame, and the counterpart to lady is lord :)

Reply:
whatever works as long as it acknowledges we have gender neutral humans